src/core/mount.c

```diff
@@ -164,6 +164,9 @@ static bool mount_needs_quota(const MountParameters *p) {
         if (p->fstype && fstype_is_network(p->fstype))
                 return false;
 
+        if (p->fstype && !fstype_needs_quota(p->fstype))
```

Comment:
the fstype_is_network() check above is now redundant, since none of the file systems listed as needing quota also are listed as being network fs.

Hence, please remove the whole network check above.